elements/gnome-sdk-core-deps/poppler.bst

```diff
@@ -0,0 +1,37 @@
+# Replaces core-deps/poppler.bst from gnome-sdk.
+
+# Because the upstream version of this is required by other elements in
+# gnome-sdk, we need to keep the existing element intact, and put this one on
+# top of it during integration by using an overlap whitelist.
+
+kind: cmake
+
+sources:
+- kind: tar
+  url: https://poppler.freedesktop.org/poppler-0.72.0.tar.xz
+  ref: c1747eb8f26e9e753c4001ed951db2896edc1021b6d0f547a0bd2a27c30ada51
+- kind: patch
+  path: files/poppler/0001-glib-Don-t-create-PopplerInputStream-with-length-0.patch
```

Comment:
This looks like an important patch. Did you try and talk to the GNOME release team? (who maintain the GNOME Sdk at [gnome-build-meta](https://gitlab.gnome.org/GNOME/gnome-build-meta/))

They might be interested in adding that patch to their element (they regularly add important patches until upstream makes a new release), at which point you can drop this entirely. :slightly_smiling_face:

Reply:
Now you mention it, yeah. That patch is from here: